Source/FakeItEasy/Core/DefaultFakeWrapperConfigurer.cs

```diff
@@ -6,21 +6,27 @@ namespace FakeItEasy.Core
     /// Handles configuring of fake objects to delegate all their calls to a wrapped instance.
     /// </summary>
     internal class DefaultFakeWrapperConfigurer
-        : IFakeWrapperConfigurer
     {
+        public DefaultFakeWrapperConfigurer(object wrappedObject)
+        {
+            this.WrappedObject = wrappedObject;
+        }
+
+        public object WrappedObject { get; private set; }
+
+        public ISelfInitializingFakeRecorder Recorder { get; set; }

```

Comment:
I'll take a look in a bit. I did wrestle a while with how to implement the wrapping/recording while introducing the "last configuration in wins" semantics. It was tough to come up with something, which may mean there isn't a better option on the horizon, or it could mean that I was just so happy to find something that worked that I stopped looking.

If it's any consolation, the mutable field isn't so much introduced as moved from `FakeOptions.SelfInitializedFakeRecorder`. :wink:

I will take a look again, and at [your concerns about the FakeOptions.Wrapper setter](https://github.com/FakeItEasy/FakeItEasy/pull/487/files#r32370991).